tests/model_serving/model_server/utils.py

```
@@ -217,3 +225,74 @@
+def verify_keda_scaledobject(
+    client: DynamicClient,
+    isvc: InferenceService,
+    runtime_name: str | None = None,
+    expected_trigger_type: str | None = None,
+    expected_query: str | None = None,
+    expected_threshold: int | None = None,
+) -> None:
+    """
+    Verify the KEDA ScaledObject.
+    """
+    scaled_object = get_isvc_keda_scaledobject(client=client, isvc=isvc, runtime_name=runtime_name)[0]
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)